include/emp/web/Div.hpp

```diff
@@ -305,6 +312,7 @@ namespace web {
     // Get a properly cast version of info.
     internal::DivInfo * Info() { return (internal::DivInfo *) info; }
     const internal::DivInfo * Info() const { return (internal::DivInfo *) info; }
+    Div(internal::DivInfo * in_info) : WidgetFacet(in_info) { ; }
```

Comment:
Add some space, an explanatory comment, and info about where design pattern is explained.